buildstockbatch/postprocessing.py

```diff
@@ -628,8 +632,17 @@ def upload_file(filepath):
         bucket = s3.Bucket(s3_bucket)
         s3key = Path(s3_prefix_output).joinpath(filepath).as_posix()
         bucket.upload_file(str(full_path), str(s3key))
+    
+    def upload_buildstock_csv(filepath):
+        full_path = buildstock_dir.joinpath(filepath)
+        s3 = boto3.resource('s3')
+        bucket = s3.Bucket(s3_bucket)
+        s3_prefix_output_new = s3_prefix_output+ '/' + 'buildstock_csv' + '/'
+        s3key = Path(s3_prefix_output_new).joinpath(filepath).as_posix()
+        bucket.upload_file(str(full_path), str(s3key))
 
     dask.compute(map(dask.delayed(upload_file), all_files))
+    dask.compute(map(dask.delayed(upload_buildstock_csv), buildstock_csv))
```

Comment:
It's best to gather all the work you want to do into one list of tasks and then send that to `dask.compute` once. You don't have to call `dask.delayed` with a `map`, it was just convenient in this case. See [dask.delayed documentation](https://docs.dask.org/en/stable/delayed.html) for more details. This change depends on `buildstock_csv` being a filename, not a list with just one item in it. 

```suggestion
    tasks = list(map(dask.delayed(upload_file), all_files))
    tasks.append(dask.delayed(upload_buildstock_csv)(buildstock_csv))
    dask.compute(tasks)
```